Makefile

```diff
@@ -12,7 +12,8 @@ LOCAL_BUILD_FLAGS ?= $(BUILD_FLAGS)
 LDFLAGS ?= -X=main.version=$(VERSION)
 DOCKERDIR = docker
 
-IMAGE ?= registry.opensource.zalan.do/acid/$(BINARY)
+BASE_IMAGE ?= alpine:latest
+IMAGE ?= ghcr.io/zalando/$(BINARY)
```

Comment:
as it is not used to ghcr build (direct Dockerfile usage), i would make it just postgres-operator for local builds